tests/bsim/bluetooth/audio/src/bap_broadcast_assistant_test.c

```diff
@@ -287,6 +307,25 @@ static void test_bass_discover(void)
 	printk("Discovery complete\n");
 }
 
+static void test_bass_read_receive_states(void)
+{
+	for (uint8_t i = 0U; i < g_recv_state_count; i++) {
+		int err;
+
+		UNSET_FLAG(flag_recv_state_read);
+		err = bt_bap_broadcast_assistant_read_recv_state(default_conn, i);
+		if (err != 0) {
+			FAIL("Failed to read receive state with idx %u: %d\n",
+			     i, err);
+			return;
+		}
+
+		WAIT_FOR_FLAG(flag_recv_state_read);
+	}
+
+	printk("Receive state read complete\n");
```

Comment:
Isn't it better to use LOG() instead of printk?

Reply:
Applies in multiple places.

Reply:
We don't have a log registered for our babblesim tests (yet). Maybe we should consider that